clock: 20
bebop101:
  initial_pose:
    x: 1.0
    y: 0
    z: 1
bebop102:
  initial_pose:
    x: -1.0
    y: 0
    z: 1
observe_field:
  dim: 3
  mesh_acc: [50,25,25] #x,theta,z
  limit: [[-1.3, 1.3],[-0.8,0.8],[0.0,0.5]] #[x[min,max], theta[min,max], z[min,max]] 
drone_field:
  dim: 1
  mesh_acc: [101] #x (should contain 0 in linspace for compress)
  limit: [[-3.0, 3.0]] #[x[min,max]] 

rviz_field:
  dim: 3
  mesh_acc: [50,25,25] #x,theta,z
  limit: [[-1.3, 1.3],[-0.8,0.8],[0.0,0.5]] #[x[min,max], theta[min,max], z[min,max]] 
sigma: 0.1